package/libs/libjson-c/Makefile

```diff
@@ -8,14 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=json-c
-PKG_VERSION:=0.12.1
+PKG_VERSION:=0.13-20171207
 PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-nodoc.tar.gz
```

Comment:
No need to change how to fetch tarball. Please refer to https://github.com/json-c/json-c/wiki#overview

Reply:
ack ; thanks for pointing it out